src/api.cc

```diff
@@ -353,7 +354,20 @@ CheckInResult AkliteClient::CheckInLocal(const LocalUpdateSource* local_update_s
   } catch (const Uptane::SecurityException& exc) {
     const std::string err{exc.what()};
     LOG_ERROR << "Failed to update the local TUF repo SecurityException: " << err;
-    return CheckInResult{CheckInResult::Status::SecurityError, "", {}};
+    if (err.find("Rollback attempt") != std::string::npos) {
+      LOG_INFO << "Checking the local TUF repo...";
+      try {
+        tuf_repo_->checkMeta();
```

Comment:
If we wanna add this code to v93 then I would remove it (the code is fine). I think we should do `checkMeta` only after verification of the additional file `install.targets.json` expiration time. It will be added in v94.

But if we are not going to add this code to v93 then this code lgtm.

Reply:
I tend towards breaking this PR into two:

1. The PR that has the TUF error specialization -  it should go to v93.
2. The PR based on the PR #1 that includes the local repo check if the offline bundle has lower version meta. This one will go to v94.

WDYT?

Reply:
Sure. If you believe it is worth to add the additional return codes now, that is fine with me.

Link on the comment is wrong, but I got the idea. Will create a separate PR for the second patch.

Reply:
sorry, I meant the PR described in the first item.

Reply:
I think it's worth adding additional error codes now because I believe the specific error messages will decrease amount of support work. Quite often customers send a request to fix an issue and the error message just says `Failed to update Image repo metadata`. Having more specific error message can help to identify the issue faster.

Reply:
Moved the second commit to a new branch, on top of the one from this PR